runtime/include/j9cfg.h.ftl

```diff
@@ -33,7 +33,9 @@ extern "C" {
 #endif
 
 #include "omrcfg.h"
+#if defined(OPENJ9_BUILD)
```

Comment:
if `openj9_version_info.h` is expected to always exist, then we shouldn't be guarding its inclusion with the `OPENJ9_BUILD` define.

I think this should be included unconditionally.

Reply:
Working on a PR to include `openj9_version_info.h` within `vendor_version.h` which is included unconditionally.

Reply:
PR has been updated, pls have another look.